bodhi/client/__init__.py

```diff
@@ -985,6 +985,90 @@ def info_release(name, url, **kwargs):
         print_release(res)
 
 
+@releases.group(name='critpath')
+def critpath_pkgs():
+    # Docs that show in the --help
+    """Interact with critical path packages of a release."""
+    # Developer Docs
+    """Manage the critical path packages of a release."""
```

Comment:
Since these are essentially the same messages, let's just document it once:

```
def critpath_pkgs():
    """Manage critical path packages."""
```